rtcdelta.py

```diff
@@ -29,22 +31,50 @@ def ensure_path_exists(path):
     if not os.path.exists(path):
         os.mkdir(path)
 
+
+def get_changes(changesets):
+    if not changesets:
+        return
+
+    for changeset in changesets:
+        print("Traversing %s" % changeset)
+        for change in changeset.getChanges():
+            print("Extracting before&after for change: %s" % change.comment)
+            change.fetchBeforeStateFile(arguments["--beforePath"])
+            change.fetchAfterStateFile(arguments["--afterPath"])
+
+
+def get_children(workitem):
```

Comment:
The name is misleading. getChildren means to get just the children. But the routine actually returns the parent WITH the children. Either change the algorithm or change the name.